src/main/scala/ir/Program.scala

```diff
@@ -172,7 +177,8 @@ class Block(
     var label: String,
     var address: Option[Int],
     var statements: ArrayBuffer[Statement],
-    var jumps: ArrayBuffer[Jump]
+    var jumps: ArrayBuffer[Jump],
+    var countOfReturnStatements: Int
```

Comment:
Wouldn't it make sense for this be in the body of the class instead of the parameters?

Reply:
Moved